veniq/dataset_collection/augmentation.py

```diff
@@ -251,42 +251,52 @@ def insert_code_with_new_file_creation(
 
     new_full_filename = Path(output_path, f'{file_name}_{method_node.name}_{invocation_node.line}.java')
     original_func = dict_original_invocations.get(invocation_node.member)[0]  # type: ignore
-    body_start_line, body_end_line = method_body_lines(original_func, file_path)
-    text_lines = read_text_with_autodetected_encoding(str(file_path)).split('\n')
-    line_to_csv = []
-    if body_start_line != body_end_line:
-        algorithm_type = determine_algorithm_insertion_type(
-            ast,
-            method_node,
-            invocation_node,
-            dict_original_invocations
-        )
-        algorithm_for_inlining = AlgorithmFactory().create_obj(algorithm_type)
-        if algorithm_type != InlineTypesAlgorithms.DO_NOTHING:
-            line_to_csv = [
-                file_path,
-                class_name,
-                text_lines[invocation_node.line - 1].lstrip(),
-                invocation_node.line,
-                original_func.line,
-                method_node.name,
-                new_full_filename,
-                body_start_line,
-                body_end_line
-            ]
-
-            algorithm_for_inlining().inline_function(
-                file_path,
-                invocation_node.line,
-                body_start_line,
-                body_end_line,
-                new_full_filename,
+    ncss = NCSSMetric().value(ast.get_subtree(original_func))
+    line_to_csv = {}
+    if ncss > 3:
+        body_start_line, body_end_line = method_body_lines(original_func, file_path)
+        text_lines = read_text_with_autodetected_encoding(str(file_path)).split('\n')
+        if body_start_line != body_end_line:
```

Comment:
can u add a comment explaining this if-condition?

Reply:
@lyriccoder  
add please
`# we do not inline one-line methods`

@KatGarmash  I'm talking about, for example:
`public String getRemainingString() { return str.substring(index); }`

Reply:
i will add it

Reply:
@Vitaly-Protasov thanks, zhenya already explained this, i just want it to be written down for the sake of documentation